internal/tracker/source/tekton/tekton.go

```diff
@@ -405,6 +425,20 @@ func PreparePackage(i *PreparePackageInput) (*hub.Package, error) {
 		},
 	}
 
+	uniqueImages := make(map[string]bool)
```

Comment:
Maybe you could use an empty `struct{}` instead of a boolean.

Reply:
Updated!